pkg/martaapi/dynamohelpers_test.go

```diff
@@ -0,0 +1,130 @@
+package martaapi_test
+
+import (
+	"io"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gstruct"
+
+	"github.com/bipol/scrapedumper/pkg/martaapi"
```

Comment:
Got some straggler references pre-organization references. Once those are consolidated I can build locally and get Travis rolling as well :grin:

Reply:
Removed the `.` dep in this file, and fixed the build issue as well